Entities/Industry/Tunnel/TunnelTravel.as

```diff
@@ -139,7 +139,7 @@ void Travel(CBlob@ this, CBlob@ caller, CBlob@ tunnel)
 		//if (isKnockable(caller) && caller.get_u8("knocked") > 0)
 		//	return;
 
-		if (caller.isAttached())   // attached - like sitting in cata? move whole cata
+		if (caller.isAttached() && getRules().gamemode_name != "CTF")   // attached - like sitting in cata? move whole cata, only in none CTF
```

Comment:
I dislike hardcoding the gamemode. What about other gamemodes like TTH and SmallCTF?